testing/renovation/platforms/vue/webpack.config.js

```diff
@@ -0,0 +1,83 @@
+const VueLoaderPlugin = require('vue-loader/lib/plugin');
+const path = require('path');
+
+module.exports = (dirname) => ({
+    mode: 'development',
+    entry: {
+        button: path.resolve(dirname, './src/button.js'),
+        buttons: path.resolve(dirname, './src/buttons.js'),
+        buttonLight: path.resolve(dirname, './src/button-light.vue')
+    },
+    resolve: {
+        alias: { vue: 'C:/Work/Devextreme/DevExtreme/testing/renovation/node_modules/vue/dist/vue.esm.js' },
```

Comment:
Is this path correct? Should we use a relative path here?
```suggestion
        alias: { vue: path.resolve(dirname, '../../node_modules/vue/dist/vue.esm.js') },
```